File: rm-backend/src/main/java/com/dwin/rm/entity/receipt/Response/ShowReceiptResponse.java
package com.dwin.rm.entity.receipt.Response;

import lombok.Builder;
import lombok.Data;

import java.util.Date;

@Data
@Builder
public class ShowReceiptResponse {
    private int receiptId;
    private String receiptName;
    private Date date;
    private Double totalAmount;
    private boolean isSettled;
}

File: du-front/DivvyUp-Web/Api/Response/ShowReceiptResponse.cs
namespace DivvyUp_Web.Api.Response
{
    public class ShowReceiptResponse
    {
        public int receiptId { get; set; }
        public string receiptName { get; set; }
        public DateTime date { get; set; }
        public double totalAmount { get; set; }
        public bool settled { get; set; }
    }
}
